api/client/src/main/java/org/projectnessie/client/http/impl/jdk11/JavaHttpClient.java

```diff
@@ -75,6 +77,26 @@ public URI getBaseUri() {
 
   @Override
   public void close() {
+    if (client != null) {
+      client
```

Comment:
Why not just call `java.net.http.HttpClient.close()` or `java.net.http.HttpClient.shutdownNow()`?

Reply:
These methods do not exist on `java.net.http.HttpClient` as far as I can tell :)

Reply:
@findepi : What JREs do you use with Trino?

Reply:
My fault. I am looking at Java 22.

Reply:
Because the implementation of `HttpClient` doesn't extend the `Closable` or have `shutdownNow` interface.